### PR TITLE
Add VOMS

### PR DESCRIPTION
**Pull request by chrisburr**

### CODE REVIEW

**Comment by chrisburr** [comment]
@conda-forge/help-c-cpp This is ready for review

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes This is ready for review

File: recipes/gsoap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{% set name = "gsoap" %}
+{% set version = "2.8.78" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/gsoap2/gsoap-2.8/gsoap_{{ version }}.zip
```
Comment (ocefpaf) [inline recipes/gsoap/meta.yaml]: The hardcoded part of the version in the URL will break the updating bot. You need to use a `split` in the jinja variable to get only the `2.8` here.

File: recipes/voms-clients/bld.bat [hunk 2]
```diff
@@ -0,0 +1,20 @@
+call mvn -Dmaven.javadoc.skip=true -Dvoms-clients.libs="%PREFIX%\share\voms-clients\lib" package
+if errorlevel 1 exit 1
+
+dir target
+7za x -so "target\voms-clients.tar.gz" | 7za x -si -aoa -ttar > NUL 2>&1
+if errorlevel 1 exit 1
+
+dir
+dir voms-clients
+dir voms-clients\bin
```
Comment (ocefpaf) [inline recipes/voms-clients/bld.bat]: I believe these are only for debugging, right?

File: recipes/voms/build.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+sed -i.bak "s#/usr/bin/soapcpp2#${PREFIX}/bin/soapcpp2#g" configure
+rm configure.bak
+
+./configure \
+    --prefix="${PREFIX}" \
+    --with-gsoap-wsdl2h="${PREFIX}/bin/wsdl2h"
+
+make -j1
+# make -j${CPU_COUNT}
```
Comment (ocefpaf) [inline recipes/voms/build.sh]: Just curious why this did not work. Also, do you have a `make check` here?

File: recipes/voms/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # Use the release candidate as it's now very old (tagged Jan 2017)
+  # Main difference is support for newer OpenSSL versions and would need to be patched regardless
+  url: https://github.com/italiangrid/{{ name }}/archive/v{{ version }}-rc0.tar.gz
```
Comment (ocefpaf) [inline recipes/voms/meta.yaml]: conda supports pep440 and version strings with `rc` in them. You should probably add that to the version variable so people installing know that this is `rc` and not a releases stable version without having to look in the recipe.

File: recipes/voms/build.sh (hunk 3 above)
Comment (chrisburr) [reply recipes/voms/build.sh]: The Makefile is buggy and doesn't declare the dependencies correctly. Fixing it isn't trivial and he build is fast enough with a single core so I gave up.

> Also, do you have a make check here?

No there isn't.

File: recipes/voms-clients/bld.bat (hunk 2 above)
Comment (chrisburr) [reply recipes/voms-clients/bld.bat]: Yes but I would rather leave them in as I haven't managed to get the Windows build working yet (I skip for now).

**Comment by chrisburr** [comment]
@ocefpaf Thanks for the quick review. I've made the changes you suggested and the CI has passed again.

**Pull request merged**